spec/unit/agent_php_spec.rb

```diff
@@ -80,8 +80,8 @@
       end
     end
 
-    it 'defines newrelic-php5enmod execute block' do
-      expect(chef_run.execute('newrelic-php5enmod')).to do_nothing
+    it 'defines newrelic-phpenmod execute block' do
+      expect(chef_run.execute('newrelic-phpenmod')).to do_nothing
```

Comment:
Maybe a good point to test could be:
```ruby
it 'defines newrelic-phpenmod execute block with php5enmod' do
 Mixlib::ShellOut.stub(:new).with('which php5enmod').and_return(shellout)
  expect(chef_run.execute('newrelic-phpenmod')).to do_nothing # or call some action.
end

it 'defines newrelic-phpenmod execute block without php5enmod' do
  shellout.stub(:error?).and_return(true)
  Mixlib::ShellOut.stub(:new).with('which php5enmod').and_return(shellout)
  expect(chef_run.execute('newrelic-phpenmod')).to do_nothing  # or call some action.
end
```
Just to be sure that the `wich php5enmod` doesn't raise any exception.